### PR TITLE
Add architecture type to the os::build::rpm::format_nvr function

### PR DESCRIPTION
**Pull request by jhadvig**
@stevekuznetsov PTAL

### CODE REVIEW

File: hack/lib/build/rpm.sh [hunk 1]
```diff
@@ -13,9 +13,11 @@
 #  - OS_RPM_NAME
 #  - OS_RPM_VERSION
 #  - OS_RPM_RELEASE
-function os::build::rpm::get_nvr_vars() {
+#  - OS_RPM_ARCHITECTURE
+function os::build::rpm::get_nvra_vars() {
 	# the package name can be overwritten but is normally 'origin'
 	OS_RPM_NAME="${OS_RPM_NAME:-"origin"}"
+	OS_RPM_ARCHITECTURE=$(uname -i)
```
Comment (stevekuznetsov) [inline hack/lib/build/rpm.sh]:
```sh
OS_RPM_ARCHITECTURE="$(uname -i)"
```

Quote

**Comment by jhadvig** [comment]
@stevekuznetsov updated

**Comment by mfojtik** [comment]
[test]

**Comment by jhadvig** [comment]
So it looks like renaming the function name causes a problem cause the test is referring to the old name

**Comment by jhadvig** [comment]
Fixed by https://github.com/openshift/aos-cd-jobs/pull/160

**Comment by mfojtik** [comment]
[merge]

**Comment by mfojtik** [comment]
After talking to @jhadvig I'm going to force-merge this in order to unblock the merge queue (without this all merges/tests will fail as the Jenkins already expecting this change).

**Pull request merged**